reactDashboard/src/components/GeoHeatMap/RegionGeoMap.jsx

```diff
@@ -62,7 +62,7 @@ const RegionGeoMap = ({
       const layer = e.target;
       const featureCode = featureCodeForFeature(layer.feature);
       const content =
-        "<strong>" + getPlaceNameByFeatureCode(featureCode) + "</strong>";
+        "<p class='region-popup'><strong>" + getPlaceNameByFeatureCode(featureCode) + "</strong></p>";
```

Comment:
It looks like there's a blank line under the place names in the tooltips now - can you remove that?

Reply:
![image](https://user-images.githubusercontent.com/22414785/95983573-c2329c00-0e19-11eb-82cf-271dd68bfb34.png)

Reply:
removed